README.md

```diff
@@ -17,7 +17,16 @@ With GoWrap you can easily add metrics, tracing, fallbacks, pools, and many othe
 
 ## Installation
 
+Install Go and run the following command.
+
+```
+# For Go 1.16+
+# Make sure that `$GOPATH/bin` is in your `$PATH`, because that's where this gets installed
```

Comment:
There's special go var called GOBIN and this is where it's actually get installed and it might be different from $GOPATH/bin. I also don't think GOPATH is somehow relevant for go 1.16+
Also nowadays people tend to install from [binaries](https://github.com/hexdigest/gowrap/releases/tag/v1.2.7) because technically you don't have to have Go installation to use gowrap.